src/main/java/com/epam/lab/paymentsystem/dao/UserDAO.java

```diff
@@ -0,0 +1,87 @@
+package com.epam.lab.paymentsystem.dao;
+
+import javax.naming.NamingException;
+import java.sql.*;
+import com.epam.lab.paymentsystem.entities.User;
+
+
+public class UserDAO implements UserDAOInterface {
+
+
+    public User createUser(User user) {
+
+        String sql = "INSERT INTO users (login, password, user_name)" + "VALUES (?, ?, ?)";
+        Connection connect = null;
+
+
+        try {
+            try {
+                connect = ConnectionPool.getConnection();
+            } catch (NamingException e) {
+                e.printStackTrace();
+            }
+            PreparedStatement ps = connect.prepareStatement(sql);
+            ps.setString(1, user.getLogin());
+            ps.setString(2, user.getPassword());
+            ps.setString(3, user.getName());
+
+            ps.executeUpdate();
+            ResultSet rs = ps.executeQuery();
+            user.setId(rs.getInt(1));
```

Comment:
You should not change the incoming data, you have to create a new user and fill it with proper values. You might as well create additional constructor\clone method for copying data. For example:
```
public User(User oldUser) {
   login = oldUser.getLogin();
   ... etc
}

OR

public static User getCopy(User old) {
   User newUser = new User();
   newUser.setLogin(old.getLogin);
   ... etc
   return newUser;
}
```